README.md

```diff
@@ -1,9 +1,30 @@
 [![Build Status](https://travis-ci.com/oasis-roles/prepare_ocp_adm.svg?branch=master)](https://travis-ci.com/oasis-roles/prepare_ocp_adm)
 
-prepare_ocp_adm
+prepare\_ocp\_adm
 ===========
 
-Basic description for prepare_ocp_adm
+This follows the [documentation](https://docs.openshift.com/container-platform/3.11/install/host_preparation.html)
+steps for deploying the [OpenShift Container Platform](https://www.openshift.com).
+This role applies specifically to a deploy host that will drive the deployment
+and yet not become part of the cluster
+
+After invoking this role, the adm host will be ready for invoking the openshift-ansible
+playbooks against the other hosts in the cluster. An invocation such as the following
+should do the trick
+
+```bash
+cd /usr/share/ansible/openshift-ansible/playbooks
+# export ANSIBLE_CONFIG=/etc/ansible/ansible.cfg
+ansible-playbook prerequisites.yml
+ansible-playbook deploy_cluster
```

Comment:
I believe "deploy_cluster" is a YAML file as well.

Reply:
True. Fixed!